.github/workflows/publish-image.yaml

```diff
@@ -0,0 +1,29 @@
+name: publish image
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'package.json'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: check if version was bumped
+      run: git diff HEAD~1 package.json | grep '"version":'
+
+    - name: login to the registry
+      run: echo ${{ secrets.REGISTRY_PASSWORD }} | docker login -u hobbyfarm --password-stdin
```

Comment:
Can we adjust username to `hobbyfarmsvc`? 
I have added the REGISTRY_PASSWORD secret.

Reply:
Might be good to set `REGISTRY_USER` or similar so we can build it in a fork. I'll add that unless you say otherwise.